lib/util/git/semantic.spec.ts

```diff
@@ -1,16 +1,13 @@
-import { RenovateConfig, getConfig, git } from '../../../test/util';
+import { RenovateConfig, git, partial } from '../../../test/util';
 import { initRepoCache } from '../cache/repository/init';
 import { detectSemanticCommits } from './semantic';
 
 jest.mock('.');
 
-let config: RenovateConfig;
+let config = partial<RenovateConfig>();
 
 beforeEach(() => {
   jest.resetAllMocks();
-  config = getConfig();
```

Comment:
you probably need to partial revert and init the config here, otherwise the tests can interfere

Reply:
Do you mean?

```ts
beforeEach(() => {
  jest.resetAllMocks();
  config = partial<RenovateConfig>({});
});
```

Or

```ts
beforeEach(() => {
  jest.resetAllMocks();
  config = getConfig();
});
```